native/common/include/jp_classbase.h

```diff
@@ -39,12 +39,12 @@ public : // JPType implementation
 
 	virtual jclass getClass() const
 	{
-		return (jclass)JPEnv::getJava()->NewLocalRef(m_Class);

```

Comment:
Local references were not needed because most resources are already held globally.